src/Image.php

```diff
@@ -1040,6 +1072,28 @@ public function writeToArray(): array
         return $result;
     }
 
+    /**
+     * @throws Exception
+     */
+    public function writeToTarget(VipsTarget $target, string $suffix, array $options = []): void
+    {
+        $filename = Utils::filenameGetFilename($suffix);
+        $string_options = Utils::filenameGetOptions($suffix);
+        $saver = FFI::vips()->vips_foreign_find_save_target($filename);
+
+        if ($saver === '') {
+            throw new Exception("can't save to target with filename $filename");
```

Comment:
How about `can't save to target with suffix $filename`? Though `filename` isn't really right here, maybe `format` would be better? 

For context, the passed-in suffix can be things like `.jpg[Q=90]` and this is split to `string_options` as `[Q=90]` and `.jpg`.